scripts/animatediff_ui.py

```diff
@@ -98,6 +98,7 @@ def __init__(self):
     def render(self, is_img2img: bool, model_dir: str):
         if not os.path.isdir(model_dir):
             os.mkdir(model_dir)
+        element_id_prefix = "ad-img2img-" if is_img2img else "ad-"
```

Comment:
better to have this format: "txt2img-ad-" or "img2img-ad-" as this is what A1111 has. You can also make this name shorter such as `elemid_prefix`